amundsen_application/static/css/_buttons-default.scss

```diff
@@ -3,8 +3,8 @@
 
   &.btn-primary,
   &.btn-default {
-    font-family: $font-family-sans-serif-bold;
-    font-weight: $font-weight-sans-serif-bold;
+    font-family: $font-family-header;
+    font-weight: $font-weight-header-regular;
```

Comment:
`$font-family-header` and `$font-weight-header-regular` like this means I've just done a global find and replace. My intent is to replace as *most* of these with one of the predefined typeography classes.

Reply:
Not sure what you mean, is this change intended or not? Can you clarify a bit more?

Reply:
We don't really have a single "bold font" and a single "regular font" anymore. We have "Roboto" for headers and "Open Sans" for everything else. Both of those fonts have variants at different weights (400, 500, 600, 700) that use different files ("-Regular", "-Semibold", "-Bold").

It didn't make sense to use the typography system builtin with Bootstrap, so I did a global find and replace for `$font-family-sans-serif-bold -> $font-family-header` and `$font-weight-sans-serif-bold -> $font-weight-header-regular` just so it would compile. I plan to revisit these rules and make adjustments accordingly.